.github/workflows/mavenCi.yml

```diff
@@ -8,9 +8,9 @@ on:
     branches: [ "master", "develop" ]
   pull_request:
     branches: [ "master", "develop" ]
-
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+  workflow_dispatch: # Allows you to run this workflow manually from the Actions tab
+  schedule:
+    - cron: "* * * * */6" #Runs pipeline once a week https://jasonet.co/posts/scheduled-actions/
```

Comment:
https://crontab.guru/#*_*_*_*_*/6

I'm not configudent that this cron string is doing what we want

Reply:
0 8 * * 1  “At 08:00 on Monday.”  should be better